ArduPlane/GCS_Mavlink.cpp

```diff
@@ -87,12 +87,12 @@ MAV_MODE GCS_MAVLINK_Plane::base_mode() const
 
 uint32_t GCS_MAVLINK_Plane::custom_mode() const
 {
-    return plane.control_mode;
+    return plane.control_mode->mode_number();
```

Comment:
A cast here will be required when the return type from `mode_number()` is corrected.

Reply:
fixed